ush/gfs_bufr.sh

```diff
@@ -82,18 +98,19 @@ case "${CASE}" in
         ${NLN} "${PARMgfs}/product/bufr_ij9km.txt"  fort.7
         ;;
     *)
-        echo "WARNING: No bufr table for this resolution, using the one for C768"
-        ${NLN} "${PARMgfs}/product/bufr_ij13km.txt" fort.7
+        echo "FATAL ERROR: Unrecognized bufr_ij*km.txt For CASE ${CASE}, ABORT!"
+        exit 1
```

Comment:
Can we keep what was here before? This will cause all our low-resolution (C48) tests to fail. What are the issues, if any, with using the 13-km list for a coarse resolution?

Reply:
If you use the 13-km list for a coarse resolution C48 run, the generated BUFR products will be inaccurate. Both the 13-km and 9-km list files contain grid location information for each sounding station. Since grid information varies with resolution, using the incorrect list file will not affect the job's running but will result in incorrect products.

Reply:
So they contain grid locations on the model coordinate system? Then why are they 13-km and 9-km instead of tied to the model resolution (e.g. bufr_ij_C768)?

Either way, if this is the case we need some for the lower resolutions too to maintain our tests.

Reply:
Yes, the files contain the nearlist grid i and j for each station, which are different for C768 and C1152. To generate the new table file for C48 resolution, I need GFS forecast files. Do you have sample files available?

Reply:
This is from the test we typically run the sounding job in (which is actually C96):
```/lfs/h2/emc/global/noscrub/walter.kolczynski/global-workflow/COMROOT/fcst_segs_ext/gfs.20211221/18/model_data/atmos/history```

Really though, we should have them for all the resolutions we support. I also feel like there should be some way to make these computations from fix files instead of needing to produce a forecast.

Reply:
Thanks, will work on this.